.circleci/config.yml

```diff
@@ -401,10 +401,15 @@ jobs:
           name: Copy over the lighthouse variables
           command: |
             cp ./monitor-ci/env.lighthouse ./monitor-ci/.env;
+      - run:
+          name: Add Ingress Host to /etc/hosts File
+          command: |
+            INGRESS_HOST=$(cat ./monitor-ci/.env | grep INGRESS_HOST= | cut -d '=' -f2);
+            grep -qxF "127.0.0.1 ${INGRESS_HOST}" /etc/hosts || sudo -- sh -c "echo \"127.0.0.1 ${INGRESS_HOST}\" >> /etc/hosts";
```

Comment:
you can just use `. ./monitor-ci/.env && grep...` to load in the environment variables instead of doing the cat thing. the `.` is the same as `source` on a mac

Reply:
yeah. I just do this for some reason 🤯